CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ CHANGELOG
 
 - Update `SummaryEvent` to include the actual name and local file path for locally-executed policy packs.
 
+- Expose resource options, parent, dependencies, and provider config to policies.
+  [#3862](https://github.com/pulumi/pulumi/pull/3862)
+
```

Comment:
Minor note that we'll need to make sure this merges correctly.

Reply:
Rebased and fixed.